cleverhans_tutorials/mnist_tutorial_cw.py

```diff
@@ -25,11 +25,16 @@
 
 FLAGS = flags.FLAGS
 
+LEARNING_RATE = .001
+CW_LEARNING_RATE = .2
+ATTACK_ITERATIONS = 100
```

Comment:
This is a nitpick but could you create flags for these (under `if __name__ == '__main__':`) so we maintain consistency with the rest of the hyper-parameters and tutorials?

Reply:
There are flags for them down below. I'm defining constants up here so that we can refer to the same constant when defining the default value for the python function and when defining the default value for the flags. Currently the defaults are hard-coded in two separate locations, and it can be confusing. While working on making the test pass, I accidentally changed the default for the python function but didn't change the default for the flag, and it took me a while to notice that my changes weren't having any effect.

Reply:
SGTM, that's a good catch. I'll make an issue so we don't forget to standardize this across the remaining tutorials